### DISCUSSION TITLE
Show the video size and support continue downloading

### DISCUSSION THREAD

**Discussion by ExHix** [Tyrrrz/YoutubeDownloader#264] [General]
I want to know how large the video file is before downloading, so that I can avoid running out of my mobile data (I use my phone to provide WiFi for my laptop) if I download some HD videos.
And sometimes the network may be unstable, causing the download easily failed, which makes me have to re-download it. I wonder if it is possible to continue downloading on the not-finished-download file instead of re-downloading the video, for re-downloading a 2GB video is really costly.

**Comment by ScottsplaysSus** [comment]
Agreed with these enhancements. Hope the developer can implement these features :)
